libs/infra-nest-server/src/lib/testServer.ts

```diff
@@ -18,15 +18,15 @@ export type TestServerOptions = {
   /**
    * Hook to override providers.
    */
-  override?: (builder: TestingModuleBuilder) => void
+  override?: (builder: TestingModuleBuilder) => TestingModuleBuilder
 }
 
 export const testServer = async (options: TestServerOptions) => {
-  const builder = Test.createTestingModule({
+  let builder = Test.createTestingModule({
     imports: [InfraModule.forRoot(options.appModule)],
   })
   if (options.override) {
-    options.override(builder)
+    builder = options.override(builder)
```

Comment:
good spot, how did the other one work 🤔

Reply:
I guess it works because it is passed the instance of the builder so modifies it in-place instead of overriding.

So this assignment is probably unnecessary, but is just logical to have it as the override method now returns a `TestingModuleBuilder` so technically the specs could create a completely new builder instance and return it.